olp-cpp-sdk-core/src/http/curl/NetworkCurl.cpp

```diff
@@ -411,9 +424,10 @@ ErrorCode NetworkCurl::SendImplementation(
     const std::shared_ptr<std::ostream>& payload,
     Network::HeaderCallback header_callback,
     Network::DataCallback data_callback, Network::Callback callback) {
-  OLP_SDK_LOG_TRACE(kLogTag, "send with id = " << id);
-
   if (!IsStarted()) {
+    OLP_SDK_LOG_ERROR(kLogTag, "can't send the request with url="
```

Comment:
Try to find shorter messages, these are just for you to find bugs.
("Send - network offline, ulr=" << request.GetUrl()); Will also do the job and is not 3 lines long.

Reply:
>> these are just for you to find bugs.
Not only for me, they are useful for other SDK users as well :)

Reply:
Most likely they will hand it over to us, if not still they are developers and they will look into the code. So the esthetical nature is not very important here. It is more important to log all the necessary information so that it can be traced back.